myhpi/urls.py

```diff
@@ -33,7 +34,20 @@
     urlpatterns += staticfiles_urlpatterns()
     urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 
+if settings.ENABLE_MAILING_LISTS:
+    urlpatterns += i18n_patterns(
+        path("lists/", include("myhpi.tenca_django.urls")),
+        # This is done manually, but shouldn't. It's all due to the ominous "problems with trailing slashes"
```

Comment:
maybe we can resolve this in the new project?

Reply:
Could also be a separate issue.

Reply:
i'm not sure if this needs fixing besides removing the slash

Reply:
Didn't work for me otherwise at least.